components/script/dom/document.rs

```diff
@@ -157,6 +157,22 @@ pub enum TouchEventResult {
     Forwarded,
 }
 
+pub enum FireMouseEventType {
+    Move,
+    Over,
+    Out,
+}
+
+impl FireMouseEventType {
+    pub fn as_str(&self) -> &str {
+        match self {
+            &FireMouseEventType::Move => "mousemove",
+            &FireMouseEventType::Over => "mouseout",
+            &FireMouseEventType::Out => "mouseover",
```

Comment:
The returned str for `::Over` and `::Out` seem to be (wrongly) swapped, no?

Reply:
Good catch!

Reply:
I'm going to file a follow up PR that fixes this.